tests/proofs/dexter/dexter-properties.md

```diff
@@ -1030,7 +1030,7 @@ proposition [dexter-emitted-ops]:
 <operations> [ Transaction DEXTER Target Amount CD ] ;; _ </operations>
 ```
 
-The following proposition `[only-dexter]` states that no one other than Dexter can emit operations whose sender is Dexter.  Note that Sends, Transfers, and MintBurns are defined to be zero for operations whose sender is not Dexter.
+The following proposition `[only-dexter]` states that no one other than Dexter can emit operations whose sender is Dexter.  Note that Sends, Transfers, and MintBurns are defined to be zero for operations whose sender is not Dexter.  Also, obviously, the state variables and XTZ balance of Dexter can be updated only by Dexter.
```

Comment:
The language that the "XTZ balance of Dexter can only be updated by Dexter" seems imprecise. Only a contract can _decrease_ its own XTZ balance --- because only it is able to send money from itself. However, conversely, only another contract can _increase_ that contract's own XTZ balance, because the money it receives must come from somewhere (excepting the case that the smart contract earns rewards through participating in the consensus process).

Reply:
good point. i slightly revised the text to better reflect my intention.